go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/VictoriaMetrics/fastcache v1.12.1
 	github.com/antithesishq/antithesis-sdk-go v0.3.8
 	github.com/ava-labs/avalanchego v1.12.1
+	github.com/ava-labs/libevm v1.13.14-0.1.0-rc.1
```

Comment:
We should keep the dependency as go-ethereum until we are able to switch to libevm (importing both go-ethereum and libevm sometimes creates linker issues)